src/_playground/Routes.js

```diff
@@ -270,6 +271,10 @@ const routes = [
     }
 ];
 
+const NoMatch = () => (
+    <Header>No match</Header>
+);
```

Comment:
this is not *quite* a 404. as, the left nav is still visible. But...

I would *consider* altering the title to say `404: not found`
I would *definitely* add a link back to home as part of a page description text.